src/components/text/index.tsx

```diff
@@ -116,16 +125,25 @@ class Text extends PureComponent<PropsTypes> {
       underline && styles.underline,
       style
     ];
+  }, [
+    propsColor,
+    modifiersColor,
+    typography,
+    backgroundColor,
+    flexStyle,
+    margins,
+    center,
+    uppercase,
+    underline,
+    style
```

Comment:
In a lot of cases, new `style` reference still gets passed to `<Text />`. There, this `useMemo` will be redundant.

Reply:
Are you planning to fix those spots as well? Just an example: https://github.com/wix/react-native-ui-lib/blob/master/src/components/avatar/index.tsx#L269